types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+     * @throw WebAPIException with error type UnknownError, if any other error occurs.
+     */
+    remove(accountId: AccountId): void;
+    /**
+     * Removes the previously registered listener.
+     *
+     * Calling this function has no effect if there is no listener with given id.
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/account.read
+     *
+     * @param accountListenerId Identifier of the listener for the account changes.
+     *
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method.
+     * @throw WebAPIException with error type UnknownError, if any other error occurs.
+     */
+    removeAccountListener(accountListenerId: number): void;
+    /**
+     * Updates an account.
+     *
+     * This method can be used only an account provider application.
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/account.write
+     *
+     * @param account Account object to update.
+     * @throw WebAPIException with error type TypeMismatchError, if any of the input parameter is not compatible with the expected type for that parameter.
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method.
+     * @throw WebAPIException with error type UnknownError, if any other error occurs.
+     */
+    update(account: Account): void;
+}
+/**
+ * The AccountProvider interface represents a service provider, such as Google, Yahoo or Vodafone.
+ */
+export interface AccountProvider {
+    /**
+     * Identifier of the account provider application.
+     */
+    readonly applicationId: ApplicationId;
+    /**
+     * Capabilities of the account provider defined in IRI format.
+     *
+     * The following predefined capabilities can be used.
+     *
+     *
+     * *   http://tizen.org/account/capability/contact - Used when the account is related to contacts
+     * *   http://tizen.org/account/capability/calendar - Used when the account is related to calendar
+     *
+     */
+    readonly capabilities: ReadonlyArray<string>;
+    /**
+     * Logical (translatable) display name.
+     */
+    readonly displayName: string;
+    /**
+     * Path to the icon representing the account provider.
+     */
+    readonly iconUri: string;
+    /**
+     * Boolean value that indicates whether multiple accounts are supported.
+     */
+    readonly isMultipleAccountSupported: boolean;
+    /**
+     * Path to the small icon representing the account provider.
+     */
+    readonly smallIconUri: string;
+}
+/**
+ * The Alarm interface is an abstract interface for alarm types.
+ */
+export class Alarm {
+    constructor();
+    /**
+     * The alarm identifier.
+     */
+    readonly id: AlarmId | null;
+}
+export interface Alarm {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.